content/syllabuses/data-sci.md

```diff
@@ -23,15 +23,15 @@ _db_id: 2
 - NCIT 07: Building an online business
 - NCIT 12: Business and technology
 - NCIT 09: Ethics and Professionalism in IT
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/general-coding-best-practices" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/automated-testing-with-pytest"%}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
 - {{% contentlink path="workshops/survey-design" %}}
-- {{% contentlink path="workshops/google-forms-like-a-boss" %}}
+- {{% contentlink path="topics/google-forms-like-a-boss" %}}
```

Comment:
Are you able to create a topic out of this? If not this change is fine and we can allocate the task to someone else